pkg/plugins/golang/v2/api.go

```diff
@@ -91,7 +91,7 @@ After the scaffold is written, api will run make on the project.
 }
 
 func (p *createAPISubcommand) BindFlags(fs *pflag.FlagSet) {
-	fs.BoolVar(&p.runMake, "make", true, "if true, run make after generating files")
+	fs.BoolVar(&p.runMake, "make", true, "if true, run `make generate` after generating files")
```

Comment:
It is breaking change for v2. So, we should not tio do that.

Reply:
Can you elaborate on your thoughts here?  At first glance, it doesn't actually quite seem breaking to me because it doesn't actually change the scaffolding at all, but I could be convinced otherwise.

Reply:
I agree that it is not changing the scaffold and its a nit. However, would be impacted if a user adds custom on the make:all to be executed in this scenario or a consumer of go/v2 via customizations (which is not the case of sdk).

However, yes agree. It is to nit scenarios to hold this one. 

/hold cancel